packages/react-components/src/components/utils/Datetime.ts

```diff
@@ -52,7 +52,11 @@ export const formatDateForChatMessage = (messageDate: Date): string => {
  * @param messageDate - date of message
  * @param currentDate - date used as offset to create the user friendly timestamp (e.g. to create 'Yesterday' instead of an absolute date)
  */
-export const formatTimestampForChatMessage = (messageDate: Date, todayDate: Date, strings: ILocaleKeys): string => {
+export const formatTimestampForChatMessage = (
```

Comment:
Not for this PR: DateTime handling across locales is complicated: https://docs.microsoft.com/en-us/dotnet/standard/globalization-localization/globalization#handle-dates-and-times

Let's file a separate work item for this?

Reply:
Sure. I filed this task https://skype.visualstudio.com/SPOOL/_workitems/edit/2484990.